### ISSUE TITLE
First and Second Year of Influence series

### ISSUE THREAD

**Issue by caleb-petry** [PardeeCenterDU/IFs-Issues-Tracking#85]
Currently, there is not a separate procedure for the first two years of the _Dependence_. _Bandwidth_, or  _Influence_ series. This raises a potential concern. How is the 3-year moving average for the first two years being handled? 

An additional but potentially related issue, why is it that a fairly large portion of first-year values are recorded as “.” if we have data in 2016? For context, the United States has four and China has 90 such values in the _Influence_ forecasts.

**Comment by caleb-petry** [comment]
Of course, this is coming from a 7.73 version:

![image](https://user-images.githubusercontent.com/7695597/137981332-f06ea503-cea3-4a54-8685-fac89bdd0147.png)

Moldova alone has values displayed for Dependence or Bandwidth. So, is it correct that the "." simply represents a value so small that it is not displayed?

If so, this is more of an initial-year transient issue. I can dig into it a bit more and report back in that case.

**Comment by caleb-petry** [comment]
I took a look at it things. The initial-year jumps are likely being caused by something in the other issue threads on here and/or arms trade, for these four specifically.

What is left to address this issue thread and produce better-looking forecasts is to have the first year forecast use two years of historical data and the second year using one. However, implementing this now would make it harder to pick up issues like the initial-year values being incorrect for those four.

To that end, we should add a switch to turn on/off the first two years using historical data versus the current approach if we were to make these proposed changes. Alternatively, we could wait to implement anything here until we have identified any other changes or issues and addressed the following:

- [x] #87 
- [x] #86 
- [x] #88 
- [ ] Identify other potential issues or changes

To make any progress in addressing the initial-year values, the first two would have to be implemented. The issue relating to Alliances and LoR only matters in later years.
